src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+let elements = document.querySelectorAll('.population');
+const total = document.querySelectorAll('.total-population');
+const average = document.querySelectorAll('.average-population');
```

Comment:
You can just write `.querySelector` w/o `...All` and you can remove [0] by total and average